dabest/_classes.py

```diff
@@ -181,7 +187,7 @@ def __init__(self, data, idx, x, y, paired, id_col, ci, resamples,
                 err = "{} is not a column in `data`. ".format(id_col)
                 raise IndexError(err)
 
-        EffectSizeDataFrame_kwargs = dict(ci=ci, is_paired=paired,
+        EffectSizeDataFrame_kwargs = dict(ci=ci, is_paired=paired, is_preceding=preceding,
```

Comment:
Please change `preceding` to `repeated_measures` and `is_preceding` to ` repeated_measures`